python/lsst/afw/display/simpleFits.cc

```diff
@@ -39,9 +39,8 @@ using namespace posix;
 #include <cstring>
 #include <cassert>
 #include <cctype>
-
+#include <any>
 #include "lsst/pex/exceptions.h"
```

Comment:
I suggest leaving a blank line between the standard library and LSST-specific includes.